msgraph-mail/go/utilities/models/attachment_collection_response.go

```diff
@@ -83,23 +107,33 @@ func (m *AttachmentCollectionResponse) Serialize(writer i878a80d2330e89d26896388
             return err
         }
     }
-    {
-        err := writer.WriteAdditionalData(m.GetAdditionalData())
-        if err != nil {
-            return err
-        }
-    }
     return nil
```

Comment:
I'm not sure why this is missing now but it's probably a bug in the generation logic, it should still be here

Reply:
@rkodev sorry I think I merged that too early. Calling attention on this comment as I think it's still a defect at this point.